src/stores/Tables/Hydrator.tsx

```diff
@@ -51,6 +55,17 @@ export const TableHydrator = ({
 
     useEffect(() => {
         if (query) {
+            // THIS IS GROSS!
+            // Our table queries update the URL and use a library to help manage that. This library
+            //  ties in with the react router which means there will be an extra render. So on most table
+            //  we do not want this and can just skip one of the updates.
+            // TODO (query param rendering)
+            // Read More: https://github.com/pbeshai/use-query-params/issues/160
+            if (skipFirstHydration.current) {
+                skipFirstHydration.current = false;
+                return;
+            }
```

Comment:
:-1: :-1: :-1: :-1: :-1: :-1: 

This is annoying but it was the quickest solution. The extra renders caused by the query param stuff ends up causing double calls on both live specs and stats - which is worse than the hack so chose the lesser of two evils.